examples/java_postgresql_customvar/README.md

```diff
@@ -107,6 +107,51 @@ and install a `stable` version.
 
 This makes the `Database` custom resource available, that the application developer will use later.
 
+### Update the Database CRD
+
+Annotations are created to describe the values that should be collected and made available by the Service Binding Request's intermediary secret.
+ 
+
+```
+oc get crd databases.postgresql.baiju.dev -o yaml
+```
+ 
+The database CRD contains the list of annotations:
+
+```
+metadata:
+  annotations:
+    servicebindingoperator.redhat.io/spec.dbName: binding:env:attribute
+    servicebindingoperator.redhat.io/status.dbConfigMap-db.host: binding:env:object:configmap
+    servicebindingoperator.redhat.io/status.dbConfigMap-db.name: binding:env:object:configmap
+    servicebindingoperator.redhat.io/status.dbConfigMap-db.password: binding:env:object:configmap
+    servicebindingoperator.redhat.io/status.dbConfigMap-db.port: binding:env:object:configmap
+    servicebindingoperator.redhat.io/status.dbConfigMap-db.user: binding:env:object:configmap
+    servicebindingoperator.redhat.io/status.dbConnectionIP: binding:env:attribute
+    servicebindingoperator.redhat.io/status.dbConnectionPort: binding:env:attribute
+    servicebindingoperator.redhat.io/status.dbCredentials-password: binding:env:object:secret
+    servicebindingoperator.redhat.io/status.dbCredentials-user: binding:env:object:secret
+    servicebindingoperator.redhat.io/status.dbName: binding:env:attribute
+
+```
+
+We can add more annotations to collect data from the fields. 
+
+To edit the CRD run the command:
+
+```
+oc edit crd databases.postgresql.baiju.dev
+```
+
+Add these annotations under `metadata.annotations` along with other annotations.
+
+```
+servicebindingoperator.redhat.io/spec.tags: binding:env:attribute
+servicebindingoperator.redhat.io/spec.userLabels.archive: binding:env:attribute
+servicebindingoperator.redhat.io/spec.secretName: binding:env:attribute
```

Comment:
The PR is about documenting mixed binding type, but we do not mention/explain this term in README. It would be very helful for reader to have such description.

Reply:
could we achieve the same effect if we add this annotations to `Database` instance later on? If yes, such instructions could be more realistic since a user typically does not have rights to edit CRDs.

Reply:
@pedjak I thought if we should expose `mixed binding` term to end-users. 
I don't think it's a good idea. 

The term `mixed binding` itself is a little confusing which sounds like different kinds of binding where as this is more about different data structures.

Reply:
+1